tools/topology/topology2/imx8-wm8960.conf

```diff
@@ -0,0 +1,128 @@
+##
+## Example topology for i.MX8MP board with 'CODEC' codec
+##
+## (source) host.N.playback -> buffer.N.1 -> volume.N.1 -> buffer.N.2 -> SAI.SAI_DAI_INDEX(sink endpoint)
+##
+
+# Include paths
```

Comment:
I added this tplg based on` volume-playback.conf`, but I saw Intel is not using this at all.
All tplg2 files are using host-copier and dai-copier and gain, instead of host, dai or volume.

So, the simple dai, host, volume are not working with IPC4?

I saw that actually the UUID for IPC4, for volume, everywhere in Intel's tplgs, is the gain component UUID - https://github.com/thesofproject/sof/blob/main/src/audio/volume/volume_uuid.h#L25 

Also, when trying to setup the PGA component, in Linux, is looking for the corresponding module in toml config file. And in toml I only see gain module.

So, should I use also gain?

Reply:
@iuliana-prodan yes host & dai components are only meant for IPC3. With IPC4, you need the host-copier/dai-copier instead. We've not tested the volume component with IPC4 in any of the Intel topologies but I believe it should work. But the gain would be a good starting for you if you want to keep things simple for now.